titus-server-master/src/main/java/com/netflix/titus/master/mesos/kubeapiserver/direct/env/AggregatingContainerEnvFactory.java

```diff
@@ -17,31 +17,17 @@
 package com.netflix.titus.master.mesos.kubeapiserver.direct.env;
 
 import java.util.Arrays;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
-
-import com.netflix.titus.api.jobmanager.model.job.Job;
-import com.netflix.titus.api.jobmanager.model.job.Task;
 
 /**
  * Aggregate container environment variables from many sources. Evaluation happens from left to right, with the
  * next item overriding entries from previous evaluations if there is a collision.
  */
-public class AggregatingContainerEnvFactory implements ContainerEnvFactory {
+public abstract class AggregatingContainerEnvFactory implements ContainerEnvFactory {
```

Comment:
Is there a value in extracting it as an abstract class?

Reply:
I will refactor and embed the initialization in the default implementation and remove this class for brevity. Thank you!